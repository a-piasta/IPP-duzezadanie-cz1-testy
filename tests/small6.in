repairRoad;kjfkglktrcljtokghyhsmjtakcf;qmiflhfuawihyrengi;-8
extendRoute;50;nwwzhpbtugix
extendRoute;50;dgpbirzbsqodgjmueymqqara
repairRoad;cmyjeuonmrxdqaxgpcpyoleoyue;qmiflhfuawihyrengi;-3
addRoad;coewpszkastflyo;wazvsbrljqshovmqjqobkplkhes;11;14
getRouteDescription;680
getRouteDescription;646
newRoute;97;kjfkglktrcljtokghyhsmjtakcf;wazvsbrljqshovmqjqobkplkhes
repairRoad;nwwzhpbtugix;cmyjeuonmrxdqaxgpcpyoleoyue;-2
getRouteDescription;97
removeRoad;coewpszkastflyo;wazvsbrljqshovmqjqobkplkhes
addRoad;kjfkglktrcljtokghyhsmjtakcf;wazvsbrljqshovmqjqobkplkhes;23;-2
newRoute;361;ngqssqdzlr;nwwzhpbtugix
extendRoute;361;qmiflhfuawihyrengi
removeRoad;kjfkglktrcljtokghyhsmjtakcf;wazvsbrljqshovmqjqobkplkhes
getRouteDescription;361
repairRoad;kjfkglktrcljtokghyhsmjtakcf;dgpbirzbsqodgjmueymqqara;11
removeRoad;coewpszkastflyo;wazvsbrljqshovmqjqobkplkhes
newRoute;818;dgpbirzbsqodgjmueymqqara;coewpszkastflyo
extendRoute;361;cmyjeuonmrxdqaxgpcpyoleoyue
removeRoad;wazvsbrljqshovmqjqobkplkhes;kjfkglktrcljtokghyhsmjtakcf
newRoute;969;dgpbirzbsqodgjmueymqqara;dgpbirzbsqodgjmueymqqara
getRouteDescription;361
repairRoad;nwwzhpbtugix;qmiflhfuawihyrengi;14
addRoad;kjfkglktrcljtokghyhsmjtakcf;nwwzhpbtugix;18;11
repairRoad;coewpszkastflyo;qmiflhfuawihyrengi;-14
removeRoad;coewpszkastflyo;wazvsbrljqshovmqjqobkplkhes
repairRoad;zilcbmbjddyd;qmiflhfuawihyrengi;8
extendRoute;818;dignrnzxajtrmechvnp
extendRoute;50;wazvsbrljqshovmqjqobkplkhes
getRouteDescription;97
addRoad;ngqssqdzlr;ngqssqdzlr;21;2
removeRoad;wazvsbrljqshovmqjqobkplkhes;kjfkglktrcljtokghyhsmjtakcf
extendRoute;50;wazvsbrljqshovmqjqobkplkhes
getRouteDescription;796
removeRoad;nwwzhpbtugix;kjfkglktrcljtokghyhsmjtakcf
extendRoute;818;qmiflhfuawihyrengi
repairRoad;cmyjeuonmrxdqaxgpcpyoleoyue;ngqssqdzlr;-11
extendRoute;818;zilcbmbjddyd
newRoute;681;nwwzhpbtugix;dgpbirzbsqodgjmueymqqara
repairRoad;zilcbmbjddyd;cmyjeuonmrxdqaxgpcpyoleoyue;-13
repairRoad;nwwzhpbtugix;kjfkglktrcljtokghyhsmjtakcf;15
newRoute;241;nwwzhpbtugix;qmiflhfuawihyrengi
newRoute;164;cmyjeuonmrxdqaxgpcpyoleoyue;qmiflhfuawihyrengi
repairRoad;zilcbmbjddyd;dignrnzxajtrmechvnp;10
getRouteDescription;818
newRoute;440;qmiflhfuawihyrengi;dignrnzxajtrmechvnp
repairRoad;nwwzhpbtugix;ngqssqdzlr;14
extendRoute;681;qmiflhfuawihyrengi
repairRoad;dgpbirzbsqodgjmueymqqara;wazvsbrljqshovmqjqobkplkhes;7
addRoad;qmiflhfuawihyrengi;dgpbirzbsqodgjmueymqqara;13;1
removeRoad;kjfkglktrcljtokghyhsmjtakcf;nwwzhpbtugix
extendRoute;361;coewpszkastflyo
extendRoute;361;dignrnzxajtrmechvnp
addRoad;nwwzhpbtugix;cmyjeuonmrxdqaxgpcpyoleoyue;29;12
repairRoad;nwwzhpbtugix;dignrnzxajtrmechvnp;7
addRoad;wazvsbrljqshovmqjqobkplkhes;zilcbmbjddyd;4;-11
extendRoute;969;kjfkglktrcljtokghyhsmjtakcf
