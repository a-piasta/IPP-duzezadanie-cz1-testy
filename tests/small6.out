1: NIE
2: NIE
3: NIE
4: NIE
5: TAK
6: 
7: 
8: NIE
9: NIE
10: 
11: TAK
12: TAK
13: NIE
14: NIE
15: TAK
16: 
17: NIE
18: NIE
19: NIE
20: NIE
21: NIE
22: NIE
23: 
24: NIE
25: TAK
26: NIE
27: NIE
28: NIE
29: NIE
30: NIE
31: 
32: NIE
33: NIE
34: NIE
35: 
36: TAK
37: NIE
38: NIE
39: NIE
40: NIE
41: NIE
42: NIE
43: NIE
44: NIE
45: NIE
46: 
47: NIE
48: NIE
49: NIE
50: NIE
51: TAK
52: NIE
53: NIE
54: NIE
55: TAK
56: NIE
57: TAK
58: NIE
